src/js/components/CheckBoxGroup/doc.js

```diff
@@ -0,0 +1,53 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils';
+
+export const doc = CheckBoxGroup => {
+  const DocumentedCheckBoxGroup = describe(CheckBoxGroup)
+    .availableAt(getAvailableAtBadge('CheckBoxGroup'))
+    .description('A group of CheckBoxes.')
+    .usage(
+      `import { CheckBoxGroup } from 'grommet';
+        <CheckBoxGroup />`,
+    )
+    .intrinsicElement('div');
+
+  DocumentedCheckBoxGroup.propTypes = {
+    checked: PropTypes.arrayOf(PropTypes.string).description(
```

Comment:
Might want to allow `PropTypes.oneOfType([PropTypes.string, PropTypes.number])` in case the valueKey maps to a number.

Reply:
Fixed.